contributed_definitions/yaml/NXmpes_peem.yaml

```diff
@@ -0,0 +1,114 @@
+category: application
+doc: |
+  This is a general application definition for a photo-electron emission or low energy electron microscopy/diffraction experiment.
+type: group
+(NXmpes)NXmpes_peem:
+  (NXentry):
+    definition:
+      \@version:
+      enumeration: [NXmpes_peem]
+    (NXmonitor):
+      exists: recommended
+    (NXinstrument):
+      (NXinsertion_device):
+          exists: optional # only when PEEM uses synchrotron light
```

Comment:
This is also present in `NXinstrument` already. Stating this here does not change or add anything, so I would suggest to remove it.
```suggestion
```